flarestack/cluster/submitter.py

```diff
@@ -437,7 +437,9 @@ def make_cluster_submission_script(self):
             "eval $(/cvmfs/icecube.opensciencegrid.org/py3-v4.1.0/setup.sh) \n"
             "export PYTHONPATH=" + DESYSubmitter.root_dir + "/ \n"
             "export FLARESTACK_SCRATCH_DIR=" + flarestack_scratch_dir + " \n"
-            "python " + fs_dir + "core/multiprocess_wrapper.py -f $1 -n $2 \n"
+            f"{sys.executable} "
```

Comment:
I think the f string thing looks messy here. I think the following is cleaner and should pass black:

```
f"{sys.executable} {fs_dir} core/multiprocess_wrapper.py -f $1 -n $2 \n"
```